src/mono/monoaotcross.proj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.Build.Traversal" DefaultTargets="BuildMonoCrossAllTargets">
```

Comment:
Why is this needed ? Why can't the cross compilers be built normally as on ios ?

Reply:
iOS assumes a 1:1 mapping between "host OS/arch" and "target OS/arch". "Build the iOS ARM64 cross compiler" _always_ means "Build the macOS x64 to iOS ARM64 cross compiler". The build pipelines and directory structures are built around this 1:1 assumption.

Right now, this would mean we can only build _one_ Android x64 cross compiler - Linux x64 (the build platform) to Android x64.

We need to decouple the entire 1:1 assumption in order to be able to build multiple hosts for the same target.